quintanalibre.org.ar/comun/etc/dropbear/authorized_keys

```diff
@@ -6,4 +6,8 @@ ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDIBva1qwYBMwQThY57+2OKUpld8BC7s3RruAj7hnhM
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDBNNT9RX2m9RP+UKyE3wgMrU6bstXxnI2K2MikIACAtCylMvowEw/ahypoI8njLLzYSmA+iIIoUWwYmJ6ZZBwsTMEH4C3z4HH+9+jsZSAXw34DpBvzejg67wfBmJZtzUch8+ZUwASM9MuM22eN/5sq+ya+UHTHWvfd095C+knG6UfRIsx7I8GrLLnagJHaJrh9LczHdOdlA+SXHZeUIDQOG2eXgUj9bcoXTxG0FUfA4YJV66uvum0Zcxt8dUYU2Nhwlks6DDtNLiY8qPWg4nhRh7Sk/g5Vh+GZBqibnuwjm0PEv+XqGnAiQWZZi9xLcsAfWCNl682L0Vfw68C4cf7V nickar@ingorian
-
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC8jbnSTO+i83zlwoSGYEvj0dFrNVbuiEW7Y+9GnbSsJ6Guh5acIeu3ySOHVqXB4+hrcPF/pKqRf0bkz2D+ccBmtFhjxFxOwlAYpBJpU+Lwgvw5+2K8s4f0Bq+grY8GMrsByk5g9R+3ykVeLqhAMLOrFRPo5P0Pb+Nwvpjk8ZRbR6S3kblAvhk7n9ejT3G8VgKlLi93Kg3ZGzuo96sLchEKXeLI208EApZ+CrqkMycAPcrpHpIee70JgL0gDuqoxbUhMkdlQNPi7Fz6XGJCp4pFmOoCXA5P+n/VhXY3x4gcIcRZoTRgzhsbHZtIVpbOZ2zhBShN3pKHL0tm/M1OiVJEi/gMG8Wh0iDlg6xdC4nJcxyliYJha6Ca1fJCsh9FAQu51PuJwPzDDsruV78LzvJxtzp39DUw7u2lYsVYmUod9xSNhL45qa88qWq2WqEzBmkLbatef/4tsjoOd3KrqoFNdQaMf8Yc1fi6zEgiID+IvZPW38iJwVKB1oLHV+7k8bINrNhXLrtrSsM/YS1x2RKP/jDVRb+jQ7ZGpsKSmJaos3OMrUunJLsJK7m0DnSnN+CC0BhbKhdyUUcojwvQqWmLaLtIGIQ3Ovx/ovmtvKVK/FqaJdrj7rOM7ETKSvZBz4DF8+D7r3eJtx4ks7u41bExJM6su1Iw5nQgo3oOh7fiOw== Gio <gio@diveni.re>
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDDmmm+KDqSsZkvgbY3Mm2Zdr+0KmVWjBPdMMktc1tmvna28xP/kUXfzy4Z0JStkCBl/9wWXe7QYPZpoyoL2i9uSrFlZoWRxvcGS/HGCnS7F33YCyoSNnojR8nSiW2KDmLLODIEUsh/btbgQXbo92hDMAgJIPl/GckQeRn2RqxUoOdOhXD1OJ0q9Zpc3wSHQfPd9i7hYztYs5+J/pFiBU/q/71B2VWCdjC8EwzdJmEEErx6h3eX7MMFITNtQtXjsgCkFUDlLQ6xI3WMmEF9ks6Bgw60C0LCsjXRC1EZtba2r26agrPKKWfjqMjcLHwgUIaot5oFjexgvv17JuyJsV6J san@bonzo
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDh9wTTSbhjNMnfZM2iiKvPmkAzXznYQUc6al59uNCtVOSEI/Z5LC+pNGmrkG8LqfKrnMpnH7/xgTKNkz+GeTlJCTWlG5pGBcB371qNMekdk2V2Kr+z2mRofuoelsEs7+aErklJwb6kGsj46vwXmEtpTQNLiFQcHvU+xCsTIlxbjkqo2zAhDk0mlz929jASaMzvQOTYKN6VCBQMy5gBUbwGIV/lXE04TOztDdFae2XjLaYp+1zEwx6IoPv8b7XEQlMP2PsTi33KO00E3wdgptJrLmTPe9RrKMY+yxxBU3Vyno4S04vr90MpK1wqq42WSwGB6J17N+SxkBx47B5VfLL7 dbellomo@newen
```

Comment:
Are we sure about this key? AFAIK he has not been in touch with the community for la long time